spec/support/teacher_sign_in.rb

```diff
@@ -0,0 +1,10 @@
+module TeacherSignIn
```

Comment:
Is it used anywhere right now? Because if it's not and it's going to be in next PRs, let the next PRs add it :)

Reply:
Not in use. Removed